docker-compose-develop.yml

```diff
@@ -3,16 +3,16 @@ services:
   develop:
     build: .
     ports:
-      - "4200:4100"
+      - "30504:30504"
```

Comment:
Did you mean to permanently change the ports here and at line 11? Just checking.

Reply:
Actually can keep the container port as is. Just needed to change the host port to 30504 to match the localstack [port](https://github.com/resource-watch/api-infrastructure/blob/production/terraform-k8s-infrastructure/modules/k8s_microservice_routing/microservices/area/main.tf#L14). Updated accordingly.